receiver/mongodbatlasreceiver/documentation.md

```diff
@@ -0,0 +1,130 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# mongoatlasreceiver
+
+## Metrics
+
+These are the metrics available for this scraper.
+
+| Name | Description | Unit | Type | Attributes |
+| ---- | ----------- | ---- | ---- | ---------- |
+| mongodbatlas.db.counts | Database feature size Aggregates MongoDB Metrics DATABASE_EXTENT_COUNT, DATABASE_VIEW_COUNT, DATABASE_COLLECTION_COUNT, DATABASE_OBJECT_COUNT, DATABASE_INDEX_COUNT | {objects} | Gauge | <ul>
```

Comment:
I'm having some trouble following these descriptions, though I'm not an atlas user.  Should the `Aggregates...` be in parenthesis or an additional sentence?  Also can it be `Aggregate of...` or something a bit clearer?

Reply:
Thanks- that's a great suggestion. I think the parentheses would be great here, but the aggregate of suggestion would make more sense in the source `metadata.yaml` file.